src/current/v23.1/goldengate.md

```diff
@@ -0,0 +1,530 @@
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           4.00
+    End of statistics.
+    ~~~
+
+## Perform bulk replication
+
+1. Keep the [Extract process](#set-up-extract-to-capture-data-from-a-source-database) running on Oracle and the [Replicat process](#set-up-replicat-to-deliver-data-to-cockroachdb) running for CockroachDB.
+1. In the source database, bulk insert some data:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    INSERT INTO OGGADM1.testtable2 (col1, col2)
+    SELECT level + 99, 'Example data'
+    FROM dual
+    CONNECT BY level <= 50000 - 99;
+    ~~~
+
+1. Run the status command on the Extract GGSCI terminal:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    stats EXTRACT EPOS
+    ~~~
+    
+    You will see an output similar to the following:
+
+    ~~~
+    Extracting from OGGADM1.TESTTABLE2 to OGGADM1.TESTTABLE2:
+    *** Total statistics since 2023-06-09 21:02:39 ***
+        Total inserts                          49903.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                       49903.00
+    *** Daily statistics since 2023-06-09 21:02:39 ***
+        Total inserts                          49903.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                       49903.00
+    *** Hourly statistics since 2023-06-09 21:02:39 ***
+        Total inserts                          49903.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                       49903.00
+    *** Latest statistics since 2023-06-09 21:02:39 ***
+        Total inserts                          49903.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                       49903.00
+    End of statistics.
+    ~~~
+
+1. Run the status command on the Replicat GGSCI terminal:
+
+    You'll notice that the number of inserts will take a few minutes to update. Once the process finishes inserting and committing to the database, it will report back the final inserts. During the process, it will show the previous state:
```

Comment:
```suggestion
    The number of inserts takes a few minutes to update. After the process finishes inserting and committing to the database, it will report back the final inserts. During the process, it will show the previous state:
```